bbp/default.nix

```diff
@@ -221,6 +221,12 @@ let
         bluepy = callPackage ./nse/bluepy {
         };
 
+        bluepy_0_11_2 = callPackage ./nse/bluepy {
```

Comment:
The current intent is not to advertise the 0_11 version, before we test it. Then we'll make it the new default.